app/contracts/user_preferences/params_contract.rb

```diff
@@ -48,6 +53,34 @@ def global_email_alerts
       end
     end
 
+    def global_setting_start_date_due_date_overdue
+      return unless stores_all_date_fields?(global_notifications)
```

Comment:
Does this mean, we do not validate any date alerts when when any of the 3 keys (`:start_date`, `:finish_date`, `:overdue`)  are missing? Do we need this?